modules/@apostrophecms/area/ui/apos/apps/AposAreas.js

```diff
@@ -23,12 +23,31 @@ export default function() {
     const data = JSON.parse(el.getAttribute('data'));
     const fieldId = el.getAttribute('data-field-id');
     const choices = JSON.parse(el.getAttribute('data-choices'));
+    const renderings = {};
+    const _docId = data._docId;
+
+    for (const widgetEl of el.querySelectorAll('[data-apos-widget]')) {
+      const _id = widgetEl.getAttribute('data-apos-widget');
+      const item = data.items.find(item => _id === item._id);
+      if (item) {
+        renderings[_id] = {
+          html: widgetEl.innerHTML,
+          parameters: {
+            _docId,
+            widget: item,
+            areaFieldId: fieldId,
+            type: item.type
+          }
+        };
+        widgetEl.remove();
+      } else {
```

Comment:
This empty `else` block pattern always grates at me. The comment is useful, but it's a confusing way to present it. Looks like something is missing.